tests/python/gaia-ui-tests/gaiatest/apps/contacts/app.py

```diff
@@ -32,7 +33,7 @@ def launch(self):
                 *self._settings_button_locator))))
 
     def switch_to_contacts_frame(self):
-        Wait(self.marionette).until(lambda m: self.apps.displayed_app.name == self.name)
+        Wait(self.marionette).until(lambda m: self.apps.displayed_app.origin_app_name == self.origin_app_name)
```

Comment:
No, you should use .origin here (and everywhere else).
So like this:
Wait(self.marionette).until(lambda m: self.apps.displayed_app.origin == self.origin)